components/raftstore-v2/src/operation/bucket.rs

```diff
@@ -304,11 +299,11 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         }
         let version = region_buckets.meta.version;
         let keys = region_buckets.meta.keys.clone();
-        // Notify followers to flush their relevant memtables
-        let peers = self.region().get_peers().to_vec();
```

Comment:
let version = region_buckets.meta.version;
let keys = region_buckets.meta.keys.clone();

can be moved below too.